app/controls/drag-drop.js

```diff
@@ -1,10 +1,7 @@
-import API from 'ipfs-api'
 import notifier from 'node-notifier'
 import {join} from 'path'
 import {getIPFS, logger} from './../init'
```

Comment:
noted, why have `const clipboard = require('clipboard')` (line 8) then? :+1: for consistency